ccm_web/client/src/utils/validation.ts

```diff
@@ -0,0 +1,46 @@
+import { string, StringSchema, ValidationError } from 'yup'
+
+// Yup: https://github.com/jquense/yup
+
+// Schemas
+
+const createExceededMessage = (fieldName: string, max: number): string => {
+  return `Value for the ${fieldName} must be ${max} characters in length or less.`
+}
+
+const createBlankMessage = (fieldName: string): string => `Value for the ${fieldName} may not be blank.`
+
+const createCanvasEntityNameSchema = (fieldName: string): StringSchema => {
+  return string().required(createBlankMessage(fieldName))
+    .max(255, ({ max }) => createExceededMessage(fieldName, max))
```

Comment:
Optional: `maxNameLength = 255` should be made as constant than getting embedded inside a function call